src/pages/welcome.tsx

```diff
@@ -471,6 +483,353 @@ const DataCheckSection = ({
   );
 };
 
+const VaccineDataSection = ({
+  incrementStep,
+  setRef,
+  canSetRef,
+  disabled,
+  vaccineRecords,
+}: {
+  incrementStep: () => void;
+  setRef: (ref: MutableRefObject<any>) => void;
+  canSetRef?: boolean;
+  disabled?: boolean;
+  vaccineRecords: VaccineRecord[];
+}) => {
+  const supabase = useSupabaseClient();
+  const { t } = useTranslation(["landing", "covid"]);
+  const user = useUser();
+
+  const sectionRef = useRef<any>();
+  useEffect(() => {
+    if (canSetRef) setTimeout(() => setRef(sectionRef), animationSpeed);
+  }, [canSetRef]);
+
+  const [loading, toggleLoading] = useToggle();
+
+  const [form, setForm] = useState({
+    lotno: "",
+    date: "",
+    administeringCenter: "",
+    provider: "",
+  });
+
+  const [vaccineData, setVaccineData] = useState<VaccineRecord[]>(
+    vaccineRecords
+  );
+
+  const validate = () => {
+    if (vaccineData.length == 0) return false;
+    return vaccineData.every((vaccine) => {
+      return (
+        vaccine.lotNo &&
+        vaccine.vaccineDate &&
+        vaccine.administeredBy &&
+        vaccine.vaccineName
+      );
+    });
+  };
+
+  const validateForm = () => {
+    if (
+      !form.lotno ||
+      !form.date ||
+      !form.administeringCenter ||
+      !form.provider
+    )
+      return false;
+    return true;
+  };
+
+  return (
+    <motion.div
+      initial={{ scale: 0.8, y: -280, opacity: 0 }}
+      animate={{ scale: 1, y: 0, opacity: 1 }}
+      exit={{ scale: 0.8, y: -280, opacity: 0 }}
+      transition={animationTransition}
+      ref={sectionRef}
+    >
+      <Section>
+        <Header
+          icon={<MaterialIcon icon="badge" allowCustomSize />}
+          text={t("welcome.vaccineData.title")}
+        />
+        <LayoutGridCols cols={6}>
+          <div className="col-span-3">
+            <Card type="stacked" appearance="outlined">
```

Comment:
Currently Dropdown Options overflow this card, and the lowest option is not visible. Try adding `className="overflow-visible"`.